resources/css/custom-variables.css

```diff
@@ -0,0 +1,18 @@
+:root {
+    /* spacing */
+    --dimension-spacing-none: 0em;
+    --dimension-spacing-xxsmall: 0.125em;
+    --dimension-spacing-xsmall: 0.250em;
+    --dimension-spacing-small: 0.500em;
```

Comment:
```suggestion
    --dimension-spacing-small: 0.5em;
```

Reply:
Just adding the inline comment to illustrate: I don't think the trailing zeros are needed.